pages/api/communities.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiHandler } from "next"
+
+import { config } from "../../config"
+import { HttpClient } from "../../lib/HttpClient"
+
+const CommunitiesHandler: NextApiHandler = async (req, res) => {
+  switch (req.method) {
+    case "GET": {
+      const allComunities = await HttpClient.get<Collibra.PagedCommunityResponse>(`${config.COLLIBRA_BASE_URL}/communities`, {
+        headers: { authorization: req.headers.authorization },
+        query: {
+          name: "equinor",
+          nameMatchMode: "ANYWHERE",
+        },
+      })
+
+      const equinorCommunity = allComunities.body?.results[0]
+
+      if (!equinorCommunity) {
+        return res.status(500).end()
```

Comment:
Why would this occur? Could it be that there are no communities? In case 404 would be correct?

Reply:
if the Equinor community doesn't exist, it would probably mean that something changed in Collibra. in that case a 404 would not be the correct return code, as what the consumer is asking for is not the Equinor community, but the sub-communities of the Equinor community.